src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table').children;
```

Comment:
Use `querySelector` to select table elements directly, instead of using `children` property. For example, use `document.querySelector('table thead')` to select the table header.